spec/observables/combineLatest-spec.js

```diff
@@ -28,4 +28,397 @@ describe('Observable.combineLatest', function () {
       done();
     });
   });
```

Comment:
The following tests are just copy-pasted from tests on the operator version of combineLatest. What do people think about this? On one hand we need to test whether the static combineLatest works, but on the other hand it's really duplication of code to-be-maintained.